packages/ckeditor5-image/ckeditor5-metadata.json

```diff
@@ -7,7 +7,70 @@
 			"docs": "features/image.html#inserting-images-via-pasting-url-into-editor",
 			"path": "src/autoimage.js",
 			"requires": [
-				"Image"
+				[
+					"ImageBlock",
+					"ImageInline"
+				]
+			]
+		},
+		{
+			"name": "Block image",
+			"className": "ImageBlock",
+			"description": "Enables support for inserting images between the blocks of text, like paragraphs or block quotes. This functionality is also provided by the main Image plugin. Note: The image feature is introduced in a granular form, implemented by separate plugins responsible for such aspects as image caption, toolbar, resize, upload or styles.",
+			"path": "src/imageblock.js",
+			"uiComponents": [
+				{
+					"type": "Button",
+					"name": "imageTextAlternative",
+					"iconPath": "",
+					"toolbars": [
+						"image.toolbar"
+					]
+				}
+			],
+			"htmlOutput": [
+				{
+					"elements": "figure",
+					"classes": "image"
+				},
+				{
+					"elements": "img",
+					"attributes": [
+						"alt",
+						"sizes",
+						"src",
+						"srcset",
+						"width"
+					]
+				}
+			]
+		},
+		{
+			"name": "Inline image",
+			"className": "ImageInline",
+			"description": "Enables support for inserting images inside the blocks of text, like paragraphs, header or block quotes. This functionality is also provided by the main Image plugin. Note: The image feature is introduced in a granular form, implemented by separate plugins responsible for such aspects as image toolbar, resize, upload or styles.",
+			"path": "src/imageinline.js",
```

Comment:
Missing "docs" property, so the `ImageInline` plugin will not have a "Documentation" link in Online Builder.